mava/configs/arch/anakin.yaml

```
@@ -1,13 +1,13 @@
 # --- Anakin config ---
 
 # --- Training ---
-num_envs: 16  # Number of vectorised environments per device.
+num_envs: 64  # Number of vectorised environments per device.
```

Spamming just to revert before merging if not aiming to update these 👀 
```suggestion
num_envs: 16  # Number of vectorised environments per device.
num_evaluation: 200  # Number of evenly spaced evaluations to perform during training.
```

---

Done.